list-publications/src/main/java/no/unit/nva/publication/query/ListPublishedPublicationsHandler.java

```diff
@@ -0,0 +1,65 @@
+package no.unit.nva.publication.query;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.publication.RequestUtil;
+import no.unit.nva.publication.model.PublicationSummary;
+import no.unit.nva.publication.service.PublicationService;
+import no.unit.nva.publication.service.impl.DynamoDBPublicationService;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.ApiGatewayHandler;
+import nva.commons.handlers.RequestInfo;
+import nva.commons.utils.Environment;
+import nva.commons.utils.JacocoGenerated;
+import org.apache.http.HttpStatus;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static nva.commons.utils.JsonUtils.objectMapper;
+
+public class ListPublishedPublicationsHandler extends ApiGatewayHandler<Void, PublishedPublicationsResponse> {
+
+    private final PublicationService publicationService;
+
+    /**
+     * Default constructor for MainHandler.
+     */
+    @JacocoGenerated
+    public ListPublishedPublicationsHandler() {
+        this(new DynamoDBPublicationService(
+                AmazonDynamoDBClientBuilder.defaultClient(),
+                objectMapper,
+                new Environment()),
+            new Environment());
+    }
+
+    /**
+     * Constructor for MainHandler.
+     *
+     * @param environment environment
+     */
+    public ListPublishedPublicationsHandler(PublicationService publicationService,
+                                            Environment environment) {
+        super(Void.class, environment, LoggerFactory.getLogger(ListPublishedPublicationsHandler.class));
+        this.publicationService = publicationService;
+    }
+
+    @Override
+    protected PublishedPublicationsResponse processInput(Void input, RequestInfo requestInfo, Context context)
+        throws ApiGatewayException {
+
+        int pageSize = RequestUtil.getPageSize(requestInfo);
+
+        logger.debug(String.format("Requested latest modified publications pagesize=%s", pageSize));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format int: pageSize (declared at line 52). This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 54. Consider formatting this data with &#39;%d&#39; instead.

Reply:
Using %d...